analyzers/src/SonarAnalyzer.Common/Rules/InsteadOfAnyBase.cs

```diff
@@ -39,13 +39,20 @@ public abstract class InsteadOfAnyBase<TSyntaxKind, TInvocationExpression> : Son
         KnownType.System_Collections_Generic_HashSet_T,
         KnownType.System_Collections_Generic_SortedSet_T);
 
+    protected static readonly ImmutableArray<KnownType> DbSetTypes = ImmutableArray.Create(
```

Comment:
do you cover all these types in the UT?

Reply:
I removed the non-generic versions, since you can't LINQ on them.
Covered the other one.

Reply:
Also added VB.NET UTs